rtape.c

```diff
@@ -0,0 +1,647 @@
+/* Copyright © 2020 Björn Victor (bjorn@victor.se) */
+/* Copyright © 2023 Lars Brinkoff <lars@nocrew.org> */
+
+/* RTAPE remote tape server. */
```

Comment:
Please add pointer(s) to documentation (of the program and the protocol)?

Reply:
I don't know of any documentation.  I mainly did what's needed to keep ITS' DUMP happy, and secondarily I checked the rtape.c in (Symbolics'?) Unix Chaosnet suite.

Reply:
Aha. Well, please write a line or two about that, so future readers understand what the code implements? (There are also clients and servers for LISPM, both in CADR and OpenGenera systems, it seems.)

Reply:
Fixed.